model/labels/labels_slicelabels.go

```diff
@@ -506,6 +506,12 @@ func (b *ScratchBuilder) Overwrite(ls *Labels) {
 	*ls = append((*ls)[:0], b.add...)
 }
 
+// UnsafeString must be used when passing to Add strings that are reusing buffers via the unsafe package.
+// For example, b.Add("__name__", labels.UnsafeString(yoloString(buf))).
+func UnsafeString(s string) string {
+	return strings.Clone(s) // We need to clone such strings for slicelabels implementation.
```

Comment:
Hm, this is not an ideal API...  it's kind of `StringCopy` but "sometimes" (for some build tags). 

I also, now when I look at it again, find `Add` and `UnsafeAddBytes` confusing. Both methods are unsafe (reusing memory), so why we cloning strings only on `UnsafeAddBytes` for `slicelabels` but we don't do that on `Add`? 🙈  

I guess because, we reuse bytes the most when we operate on bytes not on strings? 🤔